clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -188,6 +198,7 @@ public KafkaStoreIngestionService(
       ClusterInfoProvider clusterInfoProvider,
       ReadOnlyStoreRepository metadataRepo,
       ReadOnlySchemaRepository schemaRepo,
+      Optional<CompletableFuture<RoutingDataRepository>> routingRepositoryFuture,
```

Comment:
Per [style guide](https://github.com/linkedin/venice/blob/main/docs/dev_guide/style_guide.md#avoid-optionals), if we are not going to chain them with Lambda, it's fine to use `null`

Reply:
Noted, replaced the Optional with just the `CompletableFuture<RoutingDataRepository>` or `null`